diff --git a/node_modules/node-pandoc/index.js b/node_modules/node-pandoc/index.js
index 7b5e899..d87ec39 100644
--- a/node_modules/node-pandoc/index.js
+++ b/node_modules/node-pandoc/index.js
@@ -35,7 +35,7 @@ module.exports = function() {
 
   onStatCheck = function (err, stats) {
     // If src is a file or valid web URL, push the src back into args array
-    if ((stats && stats.isFile()) || isURL) {
+    if ((stats && stats.isFile()) || isURL || true) {   // TODO: Temporarly fix for multiple files
       args.unshift(src);
     }
 
